docs/installation/server-config/index.rst

```diff
@@ -53,8 +53,8 @@ For more information, see :ref:`ksql-param-reference`.
 KSQL_OPTS Environment Variable
 ------------------------------
 
-You can override KSQL server configuration parameters by using the ``KSQL_OPTS`` environment variable. For example, to
-change ``ui.enabled`` from true to false:
+You can override KSQL server configuration parameters by using the ``KSQL_OPTS`` environment variable. The properties are
```

Comment:
This sounds odd.  Perhaps @dguy can help to clarify this.

Here's my understanding, might be wrong:

- Right now, when you use KSQL_OPTS, you can pass in _anything_ that is then passed to the KSQL startup command (btw, is this true only for server or also for CLI?).
- You can use KSQL_OPTS to specify KSQL configuration settings like `ui.enabled`.  If you do, then you must prefix those settings with `-D=` because (technical implementation detail!) we are passing those settings as Java system properties to the KSQL process.

Also, I'd suggest to clarify the example command line below to help users understand how to specify more than 1 setting:

```
$ KSQL_OPTS=-Dui.enabled=false <path-to-confluent>/bin/ksql-server-start <path-to-confluent>/etc/ksql/ksql-server.properties
```

should probably be

```
$ KSQL_OPTS="-Dui.enabled=false" <path-to-confluent>/bin/ksql-server-start <path-to-confluent>/etc/ksql/ksql-server.properties
```
which makes it easier to understand how you can add more than one param (I have seen people try to set KSQL_OPTS twice, e.g. `$ KSQL_OPTS=-Dui.enabled=false KSQL_OPTS=-Dfoo=bar ...`.

Reply:
@dguy @miguno i have updated based on feedback.